{"uuid": "7765a217-ffd5-42b1-b5cf-b4125529ef28", "children": ["0eb85885-a72f-4062-91c2-07eb622a4130"], "befores": [{"name": "order", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button type=\"button\" class=\"Modal_modal__close_modified__3V5XS Modal_modal__close__TnseK\">...</button> is not clickable at point (1202, 192). Other element would receive the click: <div class=\"Modal_modal_overlay__x2ZCr\"></div>\n  (Session info: chrome=126.0.6478.183)\nStacktrace:\n0   chromedriver                        0x000000010d3a2078 chromedriver + 5169272\n1   chromedriver                        0x000000010d399f4a chromedriver + 5136202\n2   chromedriver                        0x000000010cf1636c chromedriver + 402284\n3   chromedriver                        0x000000010cf6a6b6 chromedriver + 747190\n4   chromedriver                        0x000000010cf684e8 chromedriver + 738536\n5   chromedriver                        0x000000010cf65ef7 chromedriver + 728823\n6   chromedriver                        0x000000010cf64d12 chromedriver + 724242\n7   chromedriver                        0x000000010cf58562 chromedriver + 673122\n8   chromedriver                        0x000000010cf86ab2 chromedriver + 862898\n9   chromedriver                        0x000000010cf57ed8 chromedriver + 671448\n10  chromedriver                        0x000000010cf86c6e chromedriver + 863342\n11  chromedriver                        0x000000010cfa5f57 chromedriver + 991063\n12  chromedriver                        0x000000010cf86853 chromedriver + 862291\n13  chromedriver                        0x000000010cf565c6 chromedriver + 665030\n14  chromedriver                        0x000000010cf56e4e chromedriver + 667214\n15  chromedriver                        0x000000010d364c90 chromedriver + 4918416\n16  chromedriver                        0x000000010d369c8d chromedriver + 4938893\n17  chromedriver                        0x000000010d36a365 chromedriver + 4940645\n18  chromedriver                        0x000000010d345d74 chromedriver + 4791668\n19  chromedriver                        0x000000010d36a659 chromedriver + 4941401\n20  chromedriver                        0x000000010d337544 chromedriver + 4732228\n21  chromedriver                        0x000000010d38a828 chromedriver + 5072936\n22  chromedriver                        0x000000010d38a9e7 chromedriver + 5073383\n23  chromedriver                        0x000000010d399afe chromedriver + 5135102\n24  libsystem_pthread.dylib             0x00007ff81337118b _pthread_start + 99\n25  libsystem_pthread.dylib             0x00007ff81336cae3 thread_start + 15\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/fixtures.py\", line 884, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/conftest.py\", line 111, in order\n    create_order(index_page)\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/helpers.py\", line 5, in create_order\n    index_page.click_cross_button_in_popup_window()\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/pages/index_page.py\", line 24, in click_cross_button_in_popup_window\n    self.click_by_element(IndexPageLocators.BUTTON_CROSS_IN_POPUP_WINDOW)\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/pages/base_page.py\", line 27, in click_by_element\n    WDWait(self.web_drv, timeout).until(ec.element_to_be_clickable(locator)).click()\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Добавляем ингредиент в заказ", "status": "passed", "parameters": [{"name": "index", "value": "1"}], "start": 1721429221323, "stop": 1721429221966}, {"name": "Добавляем ингредиент в заказ", "status": "passed", "parameters": [{"name": "index", "value": "4"}], "start": 1721429221966, "stop": 1721429222072}, {"name": "Клик на кнопку 'Оформить заказ'", "status": "passed", "start": 1721429222072, "stop": 1721429222131}, {"name": "Клик на крестик во всплывающем окне", "status": "passed", "start": 1721429222131, "stop": 1721429223545}, {"name": "Добавляем ингредиент в заказ", "status": "passed", "parameters": [{"name": "index", "value": "1"}], "start": 1721429223545, "stop": 1721429223668}, {"name": "Добавляем ингредиент в заказ", "status": "passed", "parameters": [{"name": "index", "value": "4"}], "start": 1721429223668, "stop": 1721429223783}, {"name": "Клик на кнопку 'Оформить заказ'", "status": "passed", "start": 1721429223783, "stop": 1721429223841}, {"name": "Клик на крестик во всплывающем окне", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button type=\"button\" class=\"Modal_modal__close_modified__3V5XS Modal_modal__close__TnseK\">...</button> is not clickable at point (1202, 192). Other element would receive the click: <div class=\"Modal_modal_overlay__x2ZCr\"></div>\n  (Session info: chrome=126.0.6478.183)\nStacktrace:\n0   chromedriver                        0x000000010d3a2078 chromedriver + 5169272\n1   chromedriver                        0x000000010d399f4a chromedriver + 5136202\n2   chromedriver                        0x000000010cf1636c chromedriver + 402284\n3   chromedriver                        0x000000010cf6a6b6 chromedriver + 747190\n4   chromedriver                        0x000000010cf684e8 chromedriver + 738536\n5   chromedriver                        0x000000010cf65ef7 chromedriver + 728823\n6   chromedriver                        0x000000010cf64d12 chromedriver + 724242\n7   chromedriver                        0x000000010cf58562 chromedriver + 673122\n8   chromedriver                        0x000000010cf86ab2 chromedriver + 862898\n9   chromedriver                        0x000000010cf57ed8 chromedriver + 671448\n10  chromedriver                        0x000000010cf86c6e chromedriver + 863342\n11  chromedriver                        0x000000010cfa5f57 chromedriver + 991063\n12  chromedriver                        0x000000010cf86853 chromedriver + 862291\n13  chromedriver                        0x000000010cf565c6 chromedriver + 665030\n14  chromedriver                        0x000000010cf56e4e chromedriver + 667214\n15  chromedriver                        0x000000010d364c90 chromedriver + 4918416\n16  chromedriver                        0x000000010d369c8d chromedriver + 4938893\n17  chromedriver                        0x000000010d36a365 chromedriver + 4940645\n18  chromedriver                        0x000000010d345d74 chromedriver + 4791668\n19  chromedriver                        0x000000010d36a659 chromedriver + 4941401\n20  chromedriver                        0x000000010d337544 chromedriver + 4732228\n21  chromedriver                        0x000000010d38a828 chromedriver + 5072936\n22  chromedriver                        0x000000010d38a9e7 chromedriver + 5073383\n23  chromedriver                        0x000000010d399afe chromedriver + 5135102\n24  libsystem_pthread.dylib             0x00007ff81337118b _pthread_start + 99\n25  libsystem_pthread.dylib             0x00007ff81336cae3 thread_start + 15\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/pages/index_page.py\", line 24, in click_cross_button_in_popup_window\n    self.click_by_element(IndexPageLocators.BUTTON_CROSS_IN_POPUP_WINDOW)\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/pages/base_page.py\", line 27, in click_by_element\n    WDWait(self.web_drv, timeout).until(ec.element_to_be_clickable(locator)).click()\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1721429223841, "stop": 1721429224947}], "start": 1721429221323, "stop": 1721429224955}], "start": 1721429221323, "stop": 1721429225079}